modules/table/src/main/java/org/apache/ignite/internal/table/MutableRowTupleAdapter.java

```diff
@@ -223,15 +232,37 @@ public MutableRowTupleAdapter(@NotNull Row row) {
 
     /** {@inheritDoc} */
     @Override public Tuple set(@NotNull String columnName, Object value) {
+        unmarshallRow();
+
+        tuple.set(columnName, value);
+
+        return this;
+    }
+
+    /**
+     * Converts immutable row to mutable tuple.
+     */
+    private void unmarshallRow() {
```

Comment:
`marshal` and `unmarshal` are spelled with one `l` at the end, see https://en.wikipedia.org/wiki/Marshalling_(computer_science).